index.html

```diff
@@ -0,0 +1,194 @@
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="recent-work-container1">
+        <div class="my-works">
+          <div class="my-work-contain">
+            <div class="my-work-card">
+              <img src="./images/ImgPlaceholder.png" alt="image" />
+              <h3 class="title">Multy-Post Stories</h3>
+              <p class="work-description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul class="prog-langs">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="recent-work-container2">
+        <div class="my-works">
+          <div class="my-work-contain">
+            <div class="my-work-card">
+              <img src="./images/ImgPlaceholder.png" alt="image" />
+              <h3 class="title">Multy-Post Stories</h3>
+              <p class="work-description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul class="prog-langs">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="recent-work-container3">
+        <div class="my-works">
+          <div class="my-work-contain">
+            <div class="my-work-card">
+              <img src="./images/ImgPlaceholder.png" alt="image" />
+              <h3 class="title">Multy-Post Stories</h3>
+              <p class="work-description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a
+                standard dummy text.
+              </p>
+              <ul class="prog-langs">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
In the index.html file, Please I suggest Putting all your sections between nav and footer into the main container as it is a professional way of using the semantic tag.